internal/server/server_test.go

```diff
@@ -355,7 +365,7 @@ func TestComposeImage(t *testing.T) {
 		}
 		response, body := tutils.PostResponseBody(t, "http://localhost:8086/api/image-builder/v1/compose", payload)
 		require.Equal(t, 400, response.StatusCode)
-		require.Contains(t, body, "Unknown UploadRequest type")
+		require.Contains(t, body, "value is not one of the allowed values")
```

Comment:
Same here, `...Error at \"/image_requests/0/upload_request/type\": value is not one of the allowed value...`, think it would be useful to include the type.